src/core/primitives/RovingFocusGroup/fragments/RovingFocusItem.tsx

```
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export interface RovingFocusItemProps {
+    children: any
+}
+
+function RovingFocusItem({children} : RovingFocusItemProps) {
```

ESLint

[error] A space is required after '{'.

(object-curly-spacing)